src/main/java/com/researchspace/model/preference/Preference.java

```diff
@@ -74,6 +74,9 @@ public enum Preference {
 	/** User enablement of Mendeley linking */
 	MENDELEY(Boolean.FALSE.toString(), SettingsType.BOOLEAN, PreferenceCategory.INTEGRATIONS, "Mendeley"),
 
+	/** User enablement of Digital Commons Data linking */
+	DIGITAL_COMMONS_DATA(Boolean.FALSE.toString(), SettingsType.BOOLEAN, PreferenceCategory.INTEGRATIONS, "Digital Commons Data"),
+
```

Comment:
That `Preference.java` class used to be needed for Apps page, but I think now it's mostly used for user preferences on 'My profile' page. Unless I'm missing something you shouldn't need to add a new preference here for DIGITAL_COMMONS_DATA - same as there is no preference added for other repositories (DATAVERSE/FIGSHARE/ZENODO etc.).

Reply:
We do need that for the integration prefs in the `IntegrationsHandlerImpl.java` on the `rspace-web` side

Reply:
Right, adding enablement of the integration as a Preference would work, but these days we're adding them as `appConfigIntegrations` i.e. rather than extending `IntegrationsHandlerImpl.booleanIntegrationPrefs` with new enum you I'd rather add app name to `IntegrationsHandlerImpl.isAppConfigIntegration()`. 

We should probably migrate the old integration-releated Preferences i.e. remove all preferences that have a category `PreferenceCategory.INTEGRATIONS`, I'm pretty sure these are all driven by app config tables anyway.

Reply:
REMOVED